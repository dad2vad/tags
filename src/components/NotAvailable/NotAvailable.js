import React, { PureComponent } from 'react';
import Icon from '../Icon';

class NotAvailable extends PureComponent {
  render() {
    return (
      <span
        data-tooltip="Not Available"
        disabled={false}
        className="button is-static is-tooltip is-fixed is-small is-danger is-outlined"
      >
        <Icon name="eye-slash" />
        <span>N / A</span>
      </span>
    );
  }
}

export default NotAvailable;
