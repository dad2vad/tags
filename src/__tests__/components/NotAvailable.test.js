import React from 'react';
import { shallow } from 'enzyme';
import NotAvailable from '../../components/NotAvailable';

describe('NotAvailable', () => {
  it('renders correctly', () => {
    const component = shallow(<NotAvailable />);
    expect(component).toMatchSnapshot();
  });
});
