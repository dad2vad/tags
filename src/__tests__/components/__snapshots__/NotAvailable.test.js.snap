// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotAvailable renders correctly 1`] = `
<span
  className="button is-static is-tooltip is-fixed is-small is-danger is-outlined"
  data-tooltip="Not Available"
  disabled={false}
>
  <Icon
    name="eye-slash"
  />
  <span>
    N / A
  </span>
</span>
`;
